src/include/OpenImageIO/simd.h

```diff
@@ -4759,7 +4759,8 @@ OIIO_FORCEINLINE vint4 blend (const vint4& a, const vint4& b, const vbool4& mask
     return _mm_or_si128 (_mm_and_si128(_mm_castps_si128(mask.simd()), b.simd()),
                          _mm_andnot_si128(_mm_castps_si128(mask.simd()), a.simd()));
 #elif OIIO_SIMD_NEON
-    return vbslq_s32 (mask.simd(), a.simd(), b.simd());
+    //return vbslq_s32 (mask.simd(), a.simd(), b.simd());
```

Comment:
you can remove the comments here

Reply:
thanks. removed the comments in the new commit